rero_ils/modules/items/cli.py

```diff
@@ -98,8 +101,11 @@ def create_circ_transactions(infile, verbose):
 
 def print_message(barcode, item_barcode, transaction_type):
     """Print confirmation message."""
-    click.echo('\t {0} created for patron {1} and item {2}'
-               .format(transaction_type, barcode, item_barcode))
+    click.echo(
+        '\t {0} created for patron {1} and item {2}'.format(
```

Comment:
not from you, but better to write 
`"{transaction_type}".format(transaction_type=transaction_type)`